xtask/src/cppdocs.rs

```diff
@@ -86,11 +86,11 @@ pub fn generate(show_warnings: bool) -> Result<(), Box<dyn std::error::Error>> {
     let enabled_features = cbindgen::EnabledFeatures {
```

Comment:
we could make a `cbindgen::EnabledFeatures::all_true()` constructor.

But no, because some of the features are still false.